libraries/c_sdk/standard/mqtt/src/iot_mqtt_serializer_deserializer_wrapper.c

```diff
@@ -45,13 +45,21 @@
 /*-----------------------------------------------------------*/
 
 /* Size of Puback packet. */
-#define MQTT_PACKET_PUBACK_SIZE      ( 4 )
+#define MQTT_PACKET_PUBACK_SIZE       ( 4 )
 
 /**
  * @brief Per the MQTT 3.1.1 spec, the largest "Remaining Length" of an MQTT
  * packet is this value.
  */
-#define MQTT_MAX_REMAINING_LENGTH    ( 268435455UL )
+#define MQTT_MAX_REMAINING_LENGTH     ( 268435455UL )
+
+/*
+ *
+ */
+#define SUBSCRIPTION_ACCEPTED_QOS0    ( 0x00 )               /**< @brief  Status code for accepted QOS0 subscription. */
+#define SUBSCRIPTION_REFUSED          ( 0x80 )               /**< @brief  Status code for refused subscription. */
+#define SUBSCRIPTION_ACCEPTED_QOS1    ( 0x01 )               /**< @brief  Status code for accepted QOS1 subscription. */
+#define SUBSCRIPTION_ACCEPTED_QOS2    ( 0x02 )               /**< @brief  Status code for accepted QOS1 subscription. */
```

Comment:
Rearrange the configs to have all the accepted together and refused later or before just for cleaner code.

Reply:
Addressed